frontend/src/metabase/components/Popover/Popover.css

```diff
@@ -177,7 +177,7 @@
   border-bottom-right-radius: 6px;
   border-bottom-left-radius: 6px;
   padding-top: 8px;
-  width: calc(100% - 2px);
+  width: 100%;
```

Comment:
the only thing I saw the extra 2px doing was leaving a slight space on the border above the "add filter" button which really bothered me 
![Uploading image.png…]()